presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running or memory-intensive queries, Presto offers
+an option to scale the resource management by using Spark, as it doesn't offer
```

Comment:
This isn't true right? This would be where you should talk about materialized exchange. PoS is what you use when you can't get the parallelism and resource management/isolation that you can get from Spark.

I would rewrite this as:

Presto on Spark makes it possible to leverage Sparks as an execution framework for Presto queries. This is useful for queries that want to run on thousands of nodes, requires 10s or 100s of terabytes of memory, and consume CPU years.

Spark provides several values adds like resource isolation, fine grained resource management, and Spark's scalable materialized exchange mechanism.

Reply:
done